README.md

```diff
@@ -20,3 +20,35 @@ To specify the table title for a machine, use the class feature of cukinia : `lo
 
 The source directory can be selected with the `-i` option. This directory shall contain all xml file to include to the test report.
 It can also include a `prerequisites.adoc` file that will be added on top of the document and a `notes.adoc` file that will appear at the end. Both files are optional.
+
+## Compliance matrix
+
+This feature only works with the id feature provided by the `-s` option.
+
+One or many compliance matrices can be added at the end of the report. These matrices are auto-generated by the program, using a csv file in entry.
+
+The csv must contain on each line a requirement and a test id. If a requirement concerns many test ids, it must appear on many lines.
+Here is an example :
+
+```
+cluster requirement,289
+security requirement,143
+security requirement,17
+security requirement,23
+stress resistance,34
+stress resistance,35
+documentation requirement,345
+```
+
+Please note that for now, all lines concerning the same requirement shall be put one after another. The format should be respected to ensure a correct output.
+
+For example, this will completely disrupt the document :
+
+```
+security requirement,143
+security requirement,17
+documentation requirement,345
+security requirement,23
+```
+
+Please be aware that, for now, all csv files in the source directory are interpreted as compliance matrix.
```

Comment:
I'm not a fan of this behavior.
One use case that has come up on my project is the need to add external manually defined tests to  the matrix. Ex: Hardware tests, Applicative tests that I don't develop but my client wants to be reminded they exist. The same could apply for instance with BP-28 tests not yet implemented, out of scope...
Could you add an option to add another type of csv files for these external tests?

Reply:
Feel free to discuss it with me.